pytensor/tensor/random/basic.py

```diff
@@ -526,6 +527,13 @@ def chisquare(df, size=None, **kwargs):
     return gamma(shape=df / 2.0, scale=2.0, size=size, **kwargs)
 
 
+def rayleigh(scale=1.0, *, size=None, **kwargs):
```

Comment:
Missing docstrings like other RandomVariables have (usually in `__call__` method)